Assets/Animal.cs

```diff
@@ -6,6 +6,7 @@ public class Animal : Enemy
 {
     private Rigidbody2D animal;
     private bool isAbducted = false;
+    private static float speed = 5f;
```

Comment:
I don't think we want to change animal speed anywhere, just player speed right?

Reply:
edit: I am dumb, rename to traktorSpeed?

Reply:
oh yeah, I was wondering what you were talking about, probs good to rename that it traktorspeed